setup.cfg

```diff
@@ -5,3 +5,6 @@ versionfile_source = distributed/_version.py
 versionfile_build = distributed/_version.py
 tag_prefix =
 parentdir_prefix = distributed-
+
+[bdist_wheel]
+universal=1
```

Comment:
What does this do exactly?  When I build I ususally do `python setup.py install sdist bdist_wheel --universal`, which seems to suffice.  Honestly I'm not sure what purpose the setup.cfg file serves here.  My knowledge of Python build tools is embarrassingly low.